### ISSUE TITLE
Constructor should be public

### ISSUE THREAD

**Issue by kykungz** [OOP2018/guessing-game-SupalukBenz#2]
Your game constructor's accessibility level should be `public`, so that it can be called across packages.
https://github.com/OOP2018/guessing-game-SupalukBenz/blob/e8934e39f74b9ae85f9c183527faaa6885a13744/src/GuessingGame.java#L15
